code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -162,6 +162,8 @@
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
 	sharpness = SHARP_DISMEMBER_EASY
 	bleed_force = BLEED_DEEP_WOUND
+	//Fuck you, *crowbars your fleshy thing
+	tool_behaviour = TOOL_CROWBAR
```

Comment:
This is incorrect. Armblades are not meant to function as crowbars. This PR doesn't actually fix the issues introduced, as it is only checking `TRAIT_DOOR_PRYER` inside of crowbar_act, which is incorrect behaviour. `TRAIT_DOOR_PRYER` needs to function for all items regardless of whether or not they are crowbars.

- Remove crowbar from armblades
- Remove crowbar from lighteater
- Move the `TRAIT_DOOR_PRYER` code on airlocks to be inside the standard attack chain and not as part of crowbar_act.